webpack/components/extensions/HostDetails/Cards/ErrataOverviewCard.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import {
+  Card,
+  CardHeader,
+  CardTitle,
+  CardBody,
+  Flex,
+  FlexItem,
+  GridItem,
+} from '@patternfly/react-core';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { propsToCamelCase } from 'foremanReact/common/helpers';
+import PropTypes from 'prop-types';
+import { ChartPie } from '@patternfly/react-charts';
+import { ErrataMapper } from '../../../../components/Errata';
+
+function HostInstallableErrata({
+  id, errataCounts,
+}) {
+  const errataTotal = errataCounts.total;
+  const errataSecurity = errataCounts.security;
+  const errataBug = errataCounts.bugfix;
+  const errataEnhance = errataCounts.enhancement;
+  const chartData = [{ x: 'security', y: errataSecurity }, { x: 'bugfix', y: errataBug }, { x: 'enhancement', y: errataEnhance }];
+  return (
+    <GridItem rowSpan={2} md={3} lg={5}>
+      <Card isHoverable>
+        <CardHeader>
+          <CardTitle>{__('Installable Errata')}</CardTitle>
+        </CardHeader>
+        <CardBody>
+          <Flex direction="column">
+            <FlexItem>
+              <a href={urlBuilder(`content_hosts/${id}/errata`, '')}>
+                {errataTotal} errata
+              </a>
+            </FlexItem>
+            <Flex flexWrap={{ lg: 'nowrap' }} direction="row">
+              <FlexItem>
+                <div className="erratachart" style={{ width: '350' }}>
+                  <ChartPie
+                    ariaDesc="Average number of pets"
+                    ariaTitle="Pie chart example"
+                    data={chartData}
+                    labels={({ datum }) => `${datum.x}: ${datum.y}`}
+                    height={300}
+                    constrainToVisibleArea
+                    padding={{
+                      bottom: 20,
+                      left: 20,
+                      right: 140,
+                      top: 20,
+                    }}
+                    width={350}
+                  />
+                </div>
+              </FlexItem>
+              <FlexItem>
+                <ErrataMapper data={chartData} id={id} />
+              </FlexItem>
+            </Flex>
+          </Flex>
+        </CardBody>
+      </Card>
+    </GridItem>
+  );
+}
+
+const ErrataOverviewCard = ({ hostDetails }) => {
+  if (hostDetails.content_facet_attributes) {
+    const hostId = hostDetails.id;
```

Comment:
Destructuring tips :)

If we have 
```js
const id = hostDetails.id;
```
we know we can destructure that like
```js
const { id } = hostDetails;
```

But if we want to rename our variable at the same time (like you do here), we can still use destructuring:

```suggestion
    const { id: hostId } = hostDetails;
```
This is the destructuring equivalent of what you wrote.

Reply:
One day I will get this Destructuring, I just need to take a few functions and practice with it.